CONTRIBUTING.md

```diff
@@ -57,97 +52,14 @@ Collaborators sign-off on a pull request by explicitly stating their approval in
 
 Exception to this process is allowed for high-priority changes that address security vulnerabilities. A shorter review period and modified sign-off process may be necessary depending on the nature of the change and severity of the issue. 
 
-## Release quality
-The guiding light, at all times, should be the long-term stability, reliability, maintainability, extensability, performance, and ultimately QUALITY as informed through current industry standards and best practices.
+## Release policy
+The guiding light, at all times, should be the long-term stability, reliability, maintainability, extensibility, performance, and ultimately quality as informed through current industry standards and best practices.
```

Comment:
nit:
"...informed through current industry..." should be "...informed by current industry..."
I would also add a comma after quality.